src/main/resources/com/ncc/aif/ontologies/README.md

```diff
@@ -0,0 +1,66 @@
+# AIF Ontologies
+
+The following is a listing of the ontologies found at AIDA-Interchange-Format/src/main/resources/com/ncc/aif/ontologies.
+Each ontology will contain a short description of what they are built from, what they are used for,
+and any other points of relevent information that might be of some use to individuals using these files.
+
+# Differences Between Ontology and OwlOntology
+
+Some ontologies have a corresponding file with a similar name such as SeedlingOntology and SeedlingOwlOntology. In these
+cases, the Owl ontology signifyies that the rdf file is compatible with strict owl, in particular using the correct syntax
+for more complex properties of classes such as RangeIncludes. The non-Owl ontology uses a simpler structure for easier extraction
+of important values.  Both the Owl and non-Owl ontology should define the same vocabulary.
+
+## AidaDomainOntologiesCommon
+
+A collection of top level classes most other ontologies reference as the highest level class.  Additionally, there are
+also a few universal classes that include important properties such as the CanHaveNumericValue class.  This ontology was
+created by individually selecting those class values and not from any other source of data.
+
+## EntityOntology
+
+A collection of Entity classes determined by the Ontology working group.  Entity referring to
+a physical being or intangible concept that either interacts or is subject to interaction.  This in combination with Events
+and Relations ontologies contain the vocabulary that will be used for defining messages post-M18 evaluation.
```

Comment:
With the new organization that I recommended in the top-level comment for this PR, you can remove the starting with sentence "This in combination with" and replace it with a similar sentence in the main "Program-Wide Ontology Files" section. Like "The Entities, Events, and Relations ontologies contain..."